src/render/template.rs

```diff
@@ -280,6 +282,132 @@ impl HelperDef for VersionCheckHelper {
     }
 }
 
+struct MathHelper {}
```

Comment:
This can just be `struct MathHelper;` (Rust is a bit redundant in this respect.)